appservice/src/main/java/nu/yona/server/subscriptions/rest/BuddyController.java

```diff
@@ -110,16 +114,31 @@
 	@PostMapping(value = "/")
 	@ResponseBody
 	public HttpEntity<BuddyResource> addBuddy(@RequestHeader(value = PASSWORD_HEADER) Optional<String> password,
-			@PathVariable UUID userId, @RequestBody PostPutBuddyDto postPutBuddy)
+			@PathVariable UUID userId, @RequestBody PostBuddyDto postBuddy)
 	{
 		try (CryptoSession cryptoSession = CryptoSession.start(password,
 				() -> userService.doPreparationsAndCheckCanAccessPrivateData(userId)))
 		{
-			return createResponse(buddyService.addBuddyToRequestingUser(userId, convertToBuddy(postPutBuddy), this::getInviteUrl),
+			return createResponse(buddyService.addBuddyToRequestingUser(userId, convertToBuddy(postBuddy), this::getInviteUrl),
 					HttpStatus.CREATED, createResourceAssembler(userId));
 		}
 	}
 
+	@PutMapping(value = "/{buddyId}")
+	@ResponseBody
+	public HttpEntity<BuddyResource> updateBuddy(@RequestHeader(value = PASSWORD_HEADER) Optional<String> password,
```

Comment:
Maybe good to add a comment here that this is only here for test purposes.

Reply:
Good point. Done.